src/App.stories.tsx

```diff
@@ -3,24 +3,10 @@ import { App } from "./App";
 import { css } from "emotion";
 
 import { mockSignIn } from "./lib/mockSignIn";
+import { user as aUser } from "./fixtures/user";
```

Comment:
As per my other comments, this abstraction might be useful in the future, possibly, but right now you could declare this mock locally to help keep the tests easy to reason through.